code/game/objects/items/weapons/storage/wallets.dm

```diff
@@ -122,3 +117,86 @@
 		new item2_type(src)
 	if(item3_type)
 		new item3_type(src)
+
+/obj/item/weapon/storage/wallet/proc/mob_icon_update()
+	if (ismob(src.loc))
+		var/mob/M = src.loc
+		M.update_inv_wear_id()
+
+/obj/item/weapon/storage/wallet/verb/flip_side()
+	set name = "Flip wallet side"
+	set category = "Object"
+	set src in usr
+	if(!istype(usr, /mob/living))
+		return
+	if (usr.stat || usr.restrained())
+		return
```

Comment:
I do not enjoy the style of repeating ourselves this much. This would probably work better as a component.

`use_check_and_message`